src/group.spec.js

```diff
@@ -25,8 +25,9 @@ describe('group', () => {
       id => xo.call('group.delete', {id})
     ))
     /*
-      The method "deleteUsers" will be executed only after the resolve
-      of the promise returned by "Promise.all".
+      we have separated the deletion of the group with the deletion of his users
+      because there is a bug in xo which makes that we need to delete the group before his users.
+      The name of the appropriate test: "delete the group with his users".
```

Comment:
> Deleting users must be done *AFTER* deleting the group because there is a race condition in xo-server which cause some users to not be properly deleted.
>
> The test “delete the group with its users” highlight this issue.